index.html

```diff
@@ -46,16 +48,177 @@ <h1>I’m Anthony <br> Glad to see you!</h1>
+                    <span class="sub-text"> 2015</span>
+                  </div>
+
+                   <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul>
+                      <li>html</li>
+                      <li>css</li>
+                     
+                      <li>javascript</li>
+                      </ul>
+                      <a  class="viewpro-btn">See Project</a>
+                    </div>
+                    <div class="card">
+                      <img src="mobile-images/works/Portfolio2.png"  alt="" class="img-center">
+                       <p class="project-title">Multi-Post
+                        Stories</p>
+
+                        <div class="card-sub-head">
+                          <span class="canopy">CANOPY</span>
+                          <i class="fa fa-fw fa-circle"></i>
+                          <span class="sub-text"> Back End Dev</span>
+                          <i class="fa fa-fw fa-circle"></i>
+                          <span class="sub-text"> 2015</span>
+                        </div>
+
+                       <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul>
+                          <li>html</li>
+                          <li>css</li>
+                         
+                          <li>javascript</li>
+                          </ul>
+                          <a  class="viewpro-btn">See Project</a>
+                      </div>
+          
+      </section>
+
+      <section class="aboutme">
+        <div class="header">
+          <h2 class="about-title">About Myself</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <a class="lets-connect" href="#">LET’S CONNECT</a>
+          <ul class="social">
+            <li>
+              <a href="#">
+                <img src="mobile-images/Github.svg"  alt="Github">
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="mobile-images/Linkedin.svg" alt="Linkedin">
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="mobile-images/peace.png" alt="Angelist">
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="mobile-images/Twitter.svg" alt="Twitter">
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="mobile-images/Vector-1.svg" alt="Medium">
+              </a>
+            </li>
+          </ul>
+          <a href="#" class="resume-button">Get my resume</a>
+        </div>
+  
+        <div class="languages">
```

Comment:
kindly there is too much space between the get my resume button and your language div. make sure to fix this so that your project conforms to that of the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848) template